code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -92,16 +92,20 @@
 		air.gases[/datum/gas/water_vapor][MOLES] -= MOLES_GAS_VISIBLE
 		. = REACTING
 
-//fire: combustion of plasma and volatile fuel (treated as hydrocarbons). creates hotspots. exothermic
-/datum/gas_reaction/fire
-	priority = -1 //fire should ALWAYS be last
-	name = "Hydrocarbon Combustion"
-	id = "fire"
+//tritium combustion: combustion of oxygen and tritium (treated as hydrocarbons). creates hotspots. exothermic
+/datum/gas_reaction/tritfire
+	priority = -1 //fire should ALWAYS be last, but tritium fires happen before plasma fires
+	name = "Tritium Combustion"
+	id = "tritfire"
 
-/datum/gas_reaction/fire/init_reqs()
-	min_requirements = list("TEMP" = FIRE_MINIMUM_TEMPERATURE_TO_EXIST) //doesn't include plasma reqs b/c of other, rarer, burning gases.
+/datum/gas_reaction/tritfire/init_reqs()
+	min_requirements = list(
+		"TEMP" = FIRE_MINIMUM_TEMPERATURE_TO_EXIST,
+		/datum/gas/tritium = MINIMUM_HEAT_CAPACITY,
```

Comment:
Is this the right constant to use here? I thought these were supposed to be mole counts, not heat capacities. Ditto 3 other locations.

Reply:
Yeah, even if it's some minimum sensible amount , make a separate define for this please.

Reply:
will do